ocaml/xapi/xapi_vm.ml

```diff
@@ -1061,7 +1061,10 @@ let import ~__context ~url ~sr ~full_restore ~force =
 let query_services ~__context ~self =
 	raise (Api_errors.Server_error(Api_errors.not_implemented, [ "query_services" ]))
 
+(* Disabled for the time being, pending a decision on what form the feature will take *)
 let assert_can_set_auto_update_drivers ~__context ~self ~value =
+	raise (Api_errors.Server_error(Api_errors.not_implemented, [ "set_auto_update_drivers" ]))
```

Comment:
You can't do this, it leads to the unsound type error. You _can_ fool the type checker by putting:

```
if true then raise exn
```

but this really bad.

Is this API call in a released version? If not, can't we just nuke it?